### PR TITLE
added styling to alerts modal and md-tabs

### PR DESCRIPTION
**Pull request by darushka**
This PR adds styling to Alerts modal and md-tabs component.

### CODE REVIEW

File: client/app/material/md-tabs.scss [hunk 1]
```diff
@@ -0,0 +1,28 @@
+md-tabs {
```
Comment (jsmylnycky) [inline client/app/material/md-tabs.scss]: Would you be able to quickly load up dashboard and make sure these styles dont interfere with the current dashboard styling please?

Comment (darushka) [inline client/app/material/md-tabs.scss]: Will do.

Comment (darushka) [inline client/app/material/md-tabs.scss]: @jsmylnycky 
Done. You can review.

**Comment by jsmylnycky** [comment]
:+1:  Thanks for fixing that @darushka

**Comment by rwrzesien** [comment]
:+1:

**Comment by darushka** [comment]
Merged squash https://github.com/peachworks/peach.css/commit/268751e15ef8c98a0c1d051ca5b564ae1acab508

**Comment by jsmylnycky** [comment]
Releasing with v0.2.7

**Pull request closed without merging**